core/drivers/crypto/caam/include/caam_desc_defines.h

```diff
@@ -427,6 +427,9 @@
 #define AS_RNG_RESEED		0x2
 #define AS_RNG_UNINSTANTIATE	0x3
 
+/* predictive resistance */
+#define OP_ALG_PR_ON            0x2
```

Comment:
This is already defined above
https://github.com/OP-TEE/optee_os/blob/6dc6fac61ed65b12a9196bd7f2d21814e73d79a4/core/drivers/crypto/caam/include/caam_desc_defines.h#L421-L423

Reply:
Dropped the commit.